declarations.d.ts

```diff
@@ -4,10 +4,109 @@ interface Object {
 
 interface IStringDictionary extends IDictionary<string> { }
 
+
+/**
+ * Describes iTunes Connect application types
+ */
+interface IitunesConnectApplicationType {
+	/**
+	 * Applications developed for iOS
+	 * @type {string}
+	 */
+	iOS: string;
+	/**
+	 * Applications developed for Mac OS
+	 * @type {string}
+	 */
+	Mac: string;
+}
+
+/**
+ * Descibes iTunes Connect applications
+ */
+interface IitunesConnectApplication {
+	/**
+	 * Unique Apple ID for each application. Automatically generated and assigned by Apple.
+	 * @type {string}
+	 */
+	adamId: string;
+	/**
+	 * No information available.
+	 * @type {number}
+	 */
+	addOnCount: number;
+	/**
+	 * The application's bundle identifier.
+	 * @type {string}
+	 */
+	bundleId: string;
+	/**
+	 * Application's name
+	 * @type {string}
+	 */
+	name: string;
+	/**
+	 * Application's stock keeping unit. User-defined unique string to keep track of the applications
+	 * @type {string}
+	 */
+	sku: string;
+	/**
+	 * Application's type
+	 * @type {IItunesConnectApplicationTypes}
+	 */
+	type: string;
+	/**
+	 * Application's current version
+	 * @type {string}
+	 */
+	version: string;
+}
+
+/**
+ * Describes data returned from querying itunes' Content Delivery api
+ */
+interface IContentDeliveryBody {
+	/**
+	 * Error object - likely present if result's Success is false.
+	 */
+	error?: Error;
+
+	/**
+	 * Query results.
+	 */
+	result: {
+		/**
+		 * A list of the user's applications.
+		 * @type {IItunesConnectApplication[]}
+		 */
+		Applications: IitunesConnectApplication[];
+		/**
+		 * Error code - likely present if Success is false.
+		 * @type {number}
+		 */
+		ErrorCode?: number;
+		/**
+		 * Error message - likely present if Success is false.
+		 * @type {string}
+		 */
+		ErrorMessage?: string;
+		/**
+		 * Error message - likely present if Success is false.
+		 * @type {string[]}
+		 */
+		Errors?: string[];
```

Comment:
`string[]`, as far as I can tell - the errors I got while testing were strings inside an array